.github/workflows/run-and-track-benchmarks-on-main.yaml

```diff
@@ -5,69 +5,68 @@ on:
     branches:
       - main
 
+permissions:
+  checks: write
+
```

Comment:
Why do we need to grant write permissions?

Reply:
https://bencher.dev/docs/how-to/github-actions/#base-branch

It is needed by the github action as indicated here